### PR TITLE
Adds user file

### PR DESCRIPTION
**Pull request by dpeshevski**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
```
Comment (meto2208) [inline index.html:10]: odlicen proekt

File: newuserfile.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+User file
```
Comment (meto2208) [inline newuserfile.txt:1]: wrong test

**Review by meto2208 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (dpeshevski) [reply index.html:10]: thanks

File: newuserfile.txt (hunk 2 above)
Comment (dpeshevski) [reply newuserfile.txt:1]: it's correct :)

**Review by meto2208 — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (emillija) [inline index.html:7]: **25.03.2021**

File: newuserfile.txt (hunk 2 above)
Comment (emillija) [reply newuserfile.txt:1]: it's not

**Review by emillija — COMMENTED** [review]
Dali uspeav da pullnam request?

File: index.html (hunk 3 above)
Comment (emillija) [reply index.html:7]: blabla

**Pull request merged**